xtests/tst-pam_shells.pamd

```diff
@@ -0,0 +1,2 @@
+#%PAM-1.0
+auth     required       pam_shells.so
```

Comment:
git warns that there is no newline at the end of this file.

Reply:
fixed